deploy/helm/stac-scaler/README.md

```diff
@@ -0,0 +1,109 @@
+| namespace | string | `pgstac` | Namespace in Kubernetes cluster to create jobs. |
+| serviceBusConnectionString | string | `nil` | Connection String of your Service Bus. |
+| processor | object | `{}` | Defines the properties of each processor that will process your input file(s) which can be either metadata in the form of JSON file or raster data in COG format. |
+
+### Processor
+
+A `processor` entry must have values as described below. Each `processor` entry defines a image to be pull from the ACR and run in the Kubernetes Cluster as Jobs through scaling.
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| labels | array | `[]` | An array of labels to be appended to the jobs and their pods. |
+| namespace | string | `nil` | Namespace created in the Service Bus that contains the Queues, Topics and Subscriptions. |
+| topicName | string | `nil` | Name of the topic to listen for scaling the jobs. |
+| subscriptionName | string | `nil` | Name of the subscription to subscribe to listen for messages for scaling the jobs. |
+| authenticationRef | string | `azure-servicebus-auth` | Name of the TriggerAuthentication in Kubernetes to use for authenticate against the Service Bus and listen to the message in the topic. |
+| pollingInterval | int | `30` | Number of seconds between polls for scaling the jobs. |
+| minReplicaCount | int | `1` | Minimum number of jobs to create during each scaling cycle. |
+| maxReplicaCount | int | `10` | Maximum number of jobs to create during each scaling cycle. |
+| successfulJobsHistoryLimit | int | `3` | Maximum number of jobs to retain after they have successfully completed. |
+| failedJobsHistoryLimit | int | `2` | Maximum number of jobs to retain after they have failed without completion. |
+| image.repository | string | `nil` | Name of the Azure Container Registry. This needs to be the full login server name with suffix such as `azurecr.io`. |
+| image.name | string | `nil` | Name of the image in the Container Registry to use for the jobs created during scaling. |
+| image.tag | string | `latest` | Name of the image tag in the Container Registry to use for hte jobs created during scaling. |
+| image.pullPolicy | string | `Always` | Policy to use for pulling images from Container Registry before creating pods. |
+| env | array | `[]` | Key - Value pairs of environment variables to be passed to the container that runs to process the files. |
+| podAnnotations | object | `{}` | Define set of arbitrary non-identifying metadata to pods. |
+| resources.limits.cpu | string | `100m` | CPU specifications for the pod. |
+| resources.limits.memory | string | `512Mi` | Memory specifications for the pod. |
+
+The following is a minimal set of required values expected in the `values-custom.yaml`:
+
+```json
+
+replicaCount: 1
+parallelism: 1
+ttlSecondsAfterFinished: 600
+activeDeadlineSeconds: 600
+
+namespace: pgstac
+serviceBusConnectionString: "service-bus-conn-string"
```

Comment:
Can this be pulled from a key vault (e.g., using [secrets store CSI driver](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-driver)) instead of specifying in plain text?

Reply:
Created a task for the next sprint to address this.